react-native-libraries.json

```
@@ -15626,5 +15626,13 @@
     "ios": true,
     "android": true,
     "newArchitecture": false
+  },
+  {
+    "githubUrl": "https://github.com/expo/expo",
+    "npmPkg": "expo-checkbox",
```

Hey @starsep, thank for adding new entry to the directory! 👍 

Let's correct the entry so it properly points to the package location in the monorepo:

```suggestion
    "githubUrl": "https://github.com/expo/expo/tree/main/packages/expo-checkbox",
```